com.basemark.BasemarkGPU.appdata.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- documentation for the appdata format: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<component>
```

Comment:
It should be `<component type="desktop">`. We disallow appdata without type specified as it implicitly disallows some appdata validation checks we care about. That's also why the build failed.

Reply:
Thanks, adding that type and <content-rating> seemed to make `org.freedesktop.appstream-glib validate` pass. Is there other tools that could be used to validate appdata.xml locally?

Reply:
Otherwise, should now be resolved with 46eec4b2b9bb84549c964d347068e772f67b8eac

Reply:
There are other tools but we only use `org.freedesktop.appstream-glib`. Thanks!